index.html

```diff
@@ -203,7 +206,84 @@ <h3>Skills</h3>
         </div>
       </section>
 
-     
+      <section class="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+
+        <form
+          action="https://formspree.io/f/xlezyjjy"
+          method="post"
+          class="form-action"
+        >
+          <input
+            type="text"
+            name="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          ></textarea>
+
+          <button type="submit">Get in touch</button>
+        </form>
+      </section>
+
+      <section class="footer">
```

Comment:
- [ ] Please add this `section` in the `footer` tag after creating the `footer` tag which is a requirement above. :+1: